pkg/checks/health.go

```diff
@@ -33,82 +34,88 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var stateMapping = map[int]string{
-	0: "unhealthy",
-	1: "healthy",
-}
+var (
+	_            Check = (*Health)(nil)
```

Comment:
Duplicate within health_test.go - witch is the better location I think

Reply:
I've moved this from the test file because placing it in the test file could potentially lead to the code compiling successfully even if the `Health` struct doesn't correctly implement the `Check` interface. It also separates the concerns, by keeping production and test code distinct.

Reply:
This should be part of the code being built, as this ensures runtime safety. We should still evaluate if we really need this compile time check, as it may be an indicator that we're adding functions to an interface which are redundant, since we're not using them in our code and we're not offering a library, where not all functions of an interface aren't used by the implementations.

Reply:
good point